src/ScopeAnalyzer.cpp

```diff
@@ -0,0 +1,81 @@
+#include "RunAnalysis.h"
+#include "ProgramRepresentation/ProgramFunction.h"
+namespace rlc_dataflow {
+
+PVAliasSet *findMatchingSet(std::list<ProgramVariable> vars, DisjointPVAliasSets aliasList) {
```

Comment:
This seems like a sensible utility method to have, but I don't think it belongs in this file. Instead, I'd put it either in `PVAliasSet` or in some utility class.

Utility methods like this also should have a comment explaining what their purpose is.